components/gc-chckbxrdio/index.json-ld

```diff
@@ -14,14 +14,23 @@
 		"en": "Checkboxes and radio buttons designed based on user experience testing",
 		"fr": "Cases à cocher et boutons radio conçu d'après des recherches d'expérience utilisateur"
 	},
-	"modified": "2018-07-12",
+	"modified": "2021-05-07",
 	"componentName": "gc-chckbxrdio",
-	"status": "provisional",
-	"todos": [
-		"Move CSS from generic provisional to locally here"
-	],
+	"status": "stable",
 	"pages": {
 		"docs": [
+			{
+				"title": "Checkboxes and radio buttons",
+				"language": "en",
+				"path": "gc-chckbxrdio-doc-en.html"
+			},
+			{
+				"title": "Cases à cocher et boutons radio",
+				"language": "fr",
+				"path": "gc-chckbxrdio-doc-fr.html"
+			}
+		],
+		"examples": [
```

Comment:
Can you add the reference to the design system, like how it is added in the legislation pages